tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html

```diff
@@ -154,6 +155,9 @@ <h2>StatVarObservations by StatVar</h2>
               <td>
                 <div></div>
               </td>
+              <td>
+                <div></div>
```

Comment:
Can you modify an input node to include observationPeriod so we can see it in the output?

Reply:
thank you for the feedback, please see:
- this summary_report [(raw html)](https://github.com/enjoythecode/import/blob/01cc744aa83b1fa4d1a3cb061bd6f7b0e1cb63a2/tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html) [(rendered html)](https://htmlpreview.github.io/?https://github.com/enjoythecode/import/blob/01cc744aa83b1fa4d1a3cb061bd6f7b0e1cb63a2/tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html) at `geoId/07`